src/css/App.css

```diff
@@ -0,0 +1,69 @@
+/** App styles */
+main {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: repeat(3, 1fr);
+}
+
+/** Field styles */
```

Comment:
I'm not sure what point is the right point to break these into their own CSS files. Should be relatively minor work when the time comes... maybe I should make a ticket to style everything? Or one ticket per component? idk